cmds/exp/watch/watch.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
```

Comment:
Add a bit of documentation, for example see https://github.com/u-root/u-root/blob/master/cmds/core/cat/cat.go#L5